llmfoundry/models/mpt/modeling_mpt.py

```diff
@@ -705,7 +707,34 @@ def fsdp_wrap_fn(self, module: nn.Module) -> bool:
 
     # Activation Checkpointing
     def activation_checkpointing_fn(self, module: nn.Module) -> bool:
-        return isinstance(module, MPTBlock)
+        if not hasattr(self.config, 'activation_checkpointing_target'
+                      ) or self.config.activation_checkpointing_target is None:
+            log.info(
+                f'activation checkpointing MPTBlock as activation_checkpointing_target is not set in model_config'
+            )
+            return isinstance(module, MPTBlock)
+        else:
+            act_ckpt_list = self.config.activation_checkpointing_target
+            if 'MPTBlock' in act_ckpt_list:
+                act_ckpt_list = ['MPTBlock']
+                warnings.warn(
```

Comment:
`log.info` instead of warning?
@dakinggg can verify if thats the correct log level.

Reply:
I think warning is appropriate, because it is something a user can correct (and probably should)